pkg/codegen/nodejs/gen.go

```diff
@@ -1463,14 +1463,14 @@ func (mod *modContext) genHeader(w io.Writer, imports []string, externalImports
 		sort.Strings(modules)
 
 		for _, module := range modules {
-			fmt.Fprintf(w, "import {")
+			fmt.Fprintf(w, "import { ")
```

Comment:
The addition of a space here is for aesthetic/ style guide? Consider being kind to the providers team, any change currently trips diffs for us to review so if not strictly necessary err on omitting, or separate PR/change.

Reply:
I'll revert. Yeah, I bet `prettier` probably adds a space when it processes the code (whenever that is). It's not at all necessary; I was just being pedantic. If it's an inconvenience at all, then it's not worth it.